common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -977,18 +977,16 @@ def test_save_problem_submitted_no_randomize(self, rerandomize):
 
     def test_check_button_name(self):
 
-        # If last attempt, button name changes to "Final Check"
-        # Just in case, we also check what happens if we have
-        # more attempts than allowed.
+        # Change tests temporarily until we rebase with edx
         attempts = random.randint(1, 10)
         module = CapaFactory.create(attempts=attempts - 1, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Final Check")
```

Comment:
LOL someone's fixing their broken tests @ahmedaljazzar. Looks like @Salomari1987 deserves a medal of honor.